ArduCopter/Parameters.cpp

```diff
@@ -339,7 +339,7 @@ const AP_Param::Info Copter::var_info[] = {
     // @Param: FRAME_TYPE
     // @DisplayName: Frame Type (+, X, V, etc)
     // @Description: Controls motor mixing for multicopters.  Not used for Tri or Traditional Helicopters.
-    // @Values: 0:Plus, 1:X, 2:V, 3:H, 4:V-Tail, 5:A-Tail, 10:Y6B, 11:Y6F, 12:BetaFlightX, 13:DJIX, 14:ClockwiseX, 15: I, 18: BetaFlightXReversed
+    // @Values: 0:Plus, 1:X, 2:V, 3:H, 4:V-Tail, 5:A-Tail, 6:PLUSREV 10:Y6B, 11:Y6F, 12:BetaFlightX, 13:DJIX, 14:ClockwiseX, 15:I, 16:NYT_PLUS 17:NYT_X 18:BetaFlightXReversed
```

Comment:
NYT_PLUS  and NYT_X  are not valid Copter frames, only work on tailsitter Quadplanes, other wise you will end up spinning with no yaw control.

Reply:
Use `@Values{something}` to limit the list here, and add heli in